pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -182,11 +196,13 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 	}
 
 	// Reconcile route
-	desiredRoute := resources.CreateKnativeRoute(kfsvc)
+	routeReconciler := knative.NewRouteReconciler(r.Client)
```

Comment:
Can we instantiate all of the reconcilers together at the top of the func? Might make it more clear what we're reconciling at first glance